robyn/openapi.py

```diff
@@ -389,25 +389,45 @@ def get_schema_object(self, parameter: str, param_type: Any) -> dict:
             list: "array",
         }
 
+        if param_type.__module__ == "datetime":
+            if param_type.__name__ == "datetime":
+                properties["type"] = "string"
+                properties["format"] = "date-time"
+                return properties
+            elif param_type.__name__ == "date":
+                properties["type"] = "string"
+                properties["format"] = "date"
+                return properties
+
         for type_name in type_mapping:
             if param_type is type_name:
                 properties["type"] = type_mapping[type_name]
                 return properties
 
-        # check for Optional type
+        # should check for Optional type first
         if param_type.__module__ == "typing":
-            properties["anyOf"] = [{"type": self.get_openapi_type(param_type.__args__[0])}, {"type": "null"}]
+            from typing import Optional, Union
```

Comment:
@Soroushsrd , thanks for the PR but we can move the imports to the top of the file, right?